subscription_manager/exceptions.py

```diff
@@ -0,0 +1,14 @@
+class MissingFieldsException(Exception):
```

Comment:
Please create parent exception for MissingFieldsException and WrongFrequencyException and InvalidSubsFieldException

so you can write
try:
    do
except GeneralSubscriptionException

Reply:
done